eb-api/src/main/java/io/engineblock/metrics/DeltaHdrHistogramReservoir.java

```diff
@@ -127,9 +109,6 @@ private synchronized Histogram getDataSinceLastSnapshotAndUpdate() {
         lastHistogram = intervalHistogram.copy();
         lastHistogram.setTag(metricName);
 
-        if (writer!=null) {
```

Comment:
This should not be removed. IntelliJ is not able to say when code is not used in late-binding scenarios.

Reply:
Sorry, I didn't realise this was being used by a script.  I've rebased and removed this commit; ready for re-review.